include/nbl/video/surface/CSurfaceVKWin32.h

```diff
@@ -0,0 +1,35 @@
+#ifndef __NBL_C_SURFACE_VK_WIN32_H_INCLUDED__
+#define __NBL_C_SURFACE_VK_WIN32_H_INCLUDED__
+
+#include <volk.h>
+#include "nbl/video/surface/ISurfaceWin32.h"
+#include "nbl/video/surface/ISurfaceVK.h"
+
+namespace nbl {
+namespace video
+{
+
+class IAPIConnection;
+
+class CSurfaceVKWin32 final : public ISurfaceWin32, public ISurfaceVK
+{
+public:
+    static core::smart_refctd_ptr<CSurfaceVKWin32> create(IAPIConnection* api, SCreationParams&& params);
+
+//private:
```

Comment:
at least protected

Reply:
why, anyway api for creating surfaces is not ready yet